### PR TITLE
mc competitive completed to review with friends

### LINKED ISSUE #48: `-play mc competitive`
Foundational functionality:

1. To start, only allow trivia to be multiple choice
2. Respond with reactions on the embed, record users response
3. Number of questions by default will be 10
4. Category will be random by default
5. Difficulty will be random by default

COMMAND: -play mc competitive
Should mimic the same game play as the -play tf competitive game mode

### PR DESCRIPTION
**Pull request by Jhector10**
close #48

### CODE REVIEW

**Review by elenirotsides — CHANGES_REQUESTED** [review]
Please add to the help commands

**Comment by Jhector10** [comment]
Changed the way we can stop the bot in the multiple choice competitive section. Can go over with group tomorrow if we want to add it to the other games as well

**Comment by elenirotsides** [comment]
image
 
If I don't answer and just try to stop it, it says I got it right

File: src/bot.js [hunk 1]
```diff
@@ -206,13 +207,210 @@ bot.on('message', async (message) => {
...
+                    .setDescription(
+                        // the meat and potatoes of the embed
+                        parseEntities(triviaData[i].question) + // the question
+                            '\n' + // added a space
+                            '\n**Choices**' + // added a space
+                            '\n' +
+                            '\n**A:** ' + parseEntities(choices[0]) + // outputs the choices from the array 'choices'
+                            '\n**B:** ' + parseEntities(choices[1]) +
+                            '\n**C:** ' + parseEntities(choices[2]) +
+                            '\n**D:** ' + parseEntities(choices[3]) +
```
Comment (elenirotsides) [inline src/bot.js:286]:
```suggestion
                            '\n🇦 ' + parseEntities(choices[0]) + // outputs the choices from the array 'choices'
                            '\n🇧 ' + parseEntities(choices[1]) +
                            '\n🇨 ' + parseEntities(choices[2]) +
                            '\n🇩 ' + parseEntities(choices[3]) +
```

**Review by elenirotsides — COMMENTED** [review]
Just an idea! You can render the emoji's in an embed too :D

**Comment by elenirotsides** [comment]
There should also be a help embed for the command `-play mc help` like there is for `-play mc help`, as well @Jhector10

**Review by elenirotsides — APPROVED** [review]
Excellent!! We can randomize everything later 👍🏻

**Pull request merged**